src/modules/login/login.html

```diff
@@ -41,7 +41,7 @@
                         </div>
                         <div class="form-group">
                             <div class="col-xs-4 col-md-offset-7 col-xs-offset-7 col-sm-offset-7 col-sm-4 col-md-4 submit">
-                                <button type="submit" class="btn btn-primary btn-lg">
+                                <button type="submit" class="btn btn-primary btn-lg" id="login-btn">
```

Comment:
As class attr is already defined with us, can you remove the id and make it a class.

`<button type="submit" class="btn btn-primary btn-lg login-btn">`